platform/metrics/platform-metrics-collector/src/main/java/ddf/metrics/collector/rrd4j/RrdJmxCollector.java

```diff
@@ -237,7 +238,12 @@ public void init() {
         LOGGER.trace("ENTERING: init() for metric {}", metricName);
 
         if (executorPool == null) {
-            executorPool = Executors.newCachedThreadPool();
+            executorPool = new ThreadPoolExecutor(0,
```

Comment:
❓ Is there a reason you changed this to manual construction of your threadpool instead of just calling Executors.newCachedThreadPool(StandardThreadFactoryBuilder.newThreadFactory("eventProcessorThread"))

Reply:
`newCachedThreadPool` will work here.